alot/buffers/search.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, ui, initialquery='', sort_order=None):
         self.dbman = ui.dbman
         self.ui = ui
         self.querystring = initialquery
+        self.querystring_list = [initialquery]
```

Comment:
why not `querystrings` instead?
It'd be great to add an inline comment explaining the situation:
"We store a list of notmuch query strings and this buffer will display the results for each query one after the other" or similar.

Reply:
Sounds good! I have renamed it and added your comment.